.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ executors:
   # step, and the saving the initial a workspace state.
   build-executor:
     docker:
-      - image: mozilla/fxa-circleci:ci-builder
+      - image: mozilla/fxa-circleci:ci-builder-node18
```

Comment:
all image name changes are temporary; I'll revert and squash once the PR's approved.